### DISCUSSION TITLE
GitHub Traffic in Repository Insights

### DISCUSSION THREAD

**Discussion by sh1nine96** [community/community#33389] [Other Feature Feedback, Questions, & Ideas]
# GitHub Traffic in Repo Insights

### 
when I see the traffic in my repositories, mostly I see that 3, 4, or 5 unique cloners, some blue-green lines, and new visitors are there. I don't know why someone would clone my repositories, they are either basic learning repositories or some academic projects. Any highlight would be really helpful.
![github-cloners](https://user-images.githubusercontent.com/98647580/190882224-7aafe077-2551-48e6-b269-bceaab8b66f8.jpg)

**Comment by radar-blip** [comment]
This answer might help you: #22069

Here is another answer about cloners and visitors:  #23048
